package/autoyast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Wed Dec  7 12:54:27 CET 2016 - schubi@suse.de
+
+- Cloning Software: install_recommended can be set by the
+  control.xml file (ay_clone_install_recommended_default)
```

Comment:
I think that, as cloning is related to AutoYaST only, you could just get rid of the `ay_` prefix.